build-tools/xaprepare/xaprepare/Application/GeneratedMonoAndroidProjitemsFile.cs

```diff
@@ -52,6 +55,7 @@ void WriteGroupApiInfo (StreamWriter sw, AndroidPlatform androidPlatform)
 			sw.WriteLine ($"      <Level>{androidPlatform.ApiLevel}</Level>");
 			sw.WriteLine ($"      <Id>{androidPlatform.PlatformID}</Id>");
 			sw.WriteLine ($"      <Stable>{androidPlatform.Stable}</Stable>");
+			sw.WriteLine ($"      <DotNet>{androidPlatform.ApiLevel >= dotnetApiLevel}</DotNet>");
```

Comment:
It might be more clear to rename this, `<DotNetSupported>` or something like that?